src/main/java/uk/gov/ida/verifyserviceprovider/services/EidasAssertionService.java

```diff
@@ -41,14 +43,16 @@ public EidasAssertionService(
             ConditionsValidator conditionsValidator,
             LevelOfAssuranceValidator levelOfAssuranceValidator,
             Optional<EidasMetadataResolverRepository> metadataResolverRepository,
-            SignatureValidatorFactory signatureValidatorFactory) {
+            SignatureValidatorFactory signatureValidatorFactory,
+            Optional<EuropeanIdentityConfiguration> europeanIdentityConfiguration) {
         super(subjectValidator, matchingDatasetUnmarshaller, mdsMapper);
         this.isEnabled = isEnabled;
         this.instantValidator = instantValidator;
         this.conditionsValidator = conditionsValidator;
         this.levelOfAssuranceValidator = levelOfAssuranceValidator;
         this.metadataResolverRepository = metadataResolverRepository;
         this.signatureValidatorFactory = signatureValidatorFactory;
+        this.europeanIdentityConfiguration = europeanIdentityConfiguration;
```

Comment:
Rather than passing in the Configuration object could we pass in the String value of hubConnectorId itself? We'd still have to wrap it as an Optional without a lot of extra rework, but it would reduce some method chaining further down.